.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+trim_trailing_whitespace = true
+
+[*.{cpp,hpp,c,h,cc,hh}]
+indent_style = space
+indent_size = 2
```

Comment:
The HotSpot style guide prescribes 2 space indentation.  The rest of the JDK not so much, and
counterexamples abound.  For example, java.base/*/native looks to be 4 space indentation
(consistently? I only spot-checked).